zulip/examples/create-user

```diff
@@ -29,7 +29,9 @@ usage = """create-user --new-email=<email address> --new-password=<password> --n
 Create a user. You must be a realm admin to use this API, and the user
 will be created in your realm.
 
-Example: create-user --site=http://localhost:9991 --user=rwbarton@example.com --new-email=jarthur@example.com --new-password=random17 --new-full-name 'J. Arthur Random' --new-short-name='jarthur'
+Example: create-user --site=http://localhost:9991 --new-email=jarthur@example.com --new-password=random17 --new-full-name 'J. Arthur Random' --new-short-name='jarthur'
```

Comment:
You don't need to include `--site` in the examples; it goes along with `--api-key` and `--user`.